package.py

```diff
@@ -807,6 +807,7 @@ def commands_step(path, commands):
                     prefix = claim.get("prefix_in_zip")
                     pip_requirements = claim.get("pip_requirements")
                     poetry_install = claim.get("poetry_install")
+                    additional_poetry_arg = claim.get("additional_poetry_args", [])
```

Comment:
Just a nit, but I'd prefer having `poetry_` as the prefix, e.g. `poetry_additional_args`, or shorter `poetry_extra_args`.

Reply:
Also, let's keep the same naming everywhere to ease finding things.